frontend/src/stories/How to add tabs to a scene.stories.mdx

```diff
@@ -0,0 +1,129 @@
+import { actionToUrl, urlToAction } from 'kea-router'
+
+import { Breadcrumb, DashboardsTabs } from '~/types'
+import { urls } from 'scenes/urls'
+
+export const DASHBOARDS_TAB_TO_NAME: Record<DashboardsTabs, string> = {
+    [DashboardsTabs.First]: 'First',
+    [DashboardsTabs.Second]: 'Second',
+}
+
+export const dashboardsLogic = kea([
+    path(['scenes', 'dashboard', 'dashboardsLogic']),
+    actions({
+        setCurrentTab: (tab: DashboardsTabs = DashboardsTabs.First) => ({ tab }),
+    }),
+    reducers({
+        currentTab: [
+            DashboardsTabs.First as DashboardsTabs,
+            {
+                setCurrentTab: (_, { tab }) => tab,
+            },
+        ],
+    }),
+    selectors(() => ({
+        breadcrumbs: [
+            // Optional if you'd like the breadcrumbs to show the current tab
+            (s) => [s.currentTab],
+            (tab): Breadcrumb[] => {
+                const breadcrumbs: Breadcrumb[] = [{ name: 'Dashboards' }]
+                breadcrumbs.push({
+                    name: DASHBOARDS_TAB_TO_NAME[tab],
+                })
+
+                return breadcrumbs
+            },
+        ],
+    })),
+    actionToUrl(({ values }) => {
+        return {
+            setCurrentTab: () => [urls.dashboards(values.currentTab)],
+        }
+    }),
+    urlToAction(({ actions, values }) => ({
+        '/dashboards/:tab': ({ tab }) => {
+            if (tab !== values.currentTab) {
+                actions.setCurrentTab(tab as DashboardsTabs)
+            }
+        },
+    })),
+])
+```
+
+5. Update `frontend/src/scenes/dashboards/Dashboards.tsx` to render the tabs:
+
+```tsx title="frontend/src/scenes/dashboards/Dashboards.tsx"
+import { useValues } from 'kea'
+import { router } from 'kea-router'
+import { DashboardsTabs } from '~/types'
+import { SceneExport } from 'scenes/sceneTypes'
+import { PageHeader } from 'lib/components/PageHeader'
+import { LemonTabs } from 'lib/lemon-ui/LemonTabs/LemonTabs'
+import { urls } from 'scenes/urls'
+import { dashboardsLogic, humanFriendlyTabName } from './dashboardsLogic'
+
+const DASHBOARDS_TAB_TO_CONTENT: Record<DashboardsTabs, JSX.Element> = {
+    [DashboardsTabs.First]: <div>First tab content</div>,
+    [DashboardsTabs.Second]: <div>Second tab content</div>,
+}
+
+export const scene: SceneExport = {
+    component: Dashboards,
+    logic: dashboardsLogic,
+}
+
+export function Dashboards(): JSX.Element {
+    const { currentTab } = useValues(dashboardsLogic)
+
+    return (
+        <div>
+            <PageHeader title="Dashboards" />
+            <LemonTabs
+                activeKey={currentTab}
+                onChange={(tab) => router.actions.push(urls.dashboards(tab as DashboardsTabs))}
+                tabs={Object.values(DashboardsTabs).map((tab) => ({
+                    label: DASHBOARDS_TAB_TO_NAME[tab],
+                    key: tab,
+                    content: DASHBOARDS_TAB_TO_CONTENT[tab],
```

Comment:
For non-advanced use cases it's better for the content to be part of the tab's definition

Reply:
could you bring me examples of when it's advanced vs not?

Reply:
Basically if you want to put some extra UI between the tab bar and tab contents. Otherwise, it's simpler to keep everything together,